src/main/resources/org/jenkinsci/plugins/credentialsbinding/impl/Messages.properties

```diff
@@ -1,5 +1,6 @@
 FileBinding.secret_file=Secret file
 SecretBuildWrapper.use_secret_text_s_or_file_s_=Use secret text(s) or file(s)
 StringBinding.secret_text=Secret text
-UsernamePasswordBinding.username_and_password=Username and password
+UsernamePasswordBinding.username_and_password=Username and password (conjoined)
+UsernamePasswordMultiBinding.username_and_password=Username and password (separated)
```

Comment:
Would prefer to deprecate `UsernamePasswordBinding` but it cannot simply be replaced with a `UsernamePasswordMultiBinding`, and AFAIK we cannot remove the option to add a _new_ `UsernamePasswordBinding` without also breaking reconfiguration of _existing_ jobs, so for now just leaving both options.